java/src/jmri/jmrit/beantable/LogixTableBundle_ca.properties

```diff
@@ -85,272 +85,272 @@ Unknown = Desconegut
 #rowAbrev = R
 
 LabelLogicType = Operador L\u00f2gic
-TypeLogicHint = Choose logic operator joining state variables for the antecedent ('if' part) of this conditional
-LabelAntecedent = Antecedent Expression (the 'if' part of the Conditional)
-LabelAntecedentHint = Insert parenthesis so the statement satisfies the logic you want
-#HelpButton = Help
-LogicHelpText1 = Mixing AND's and OR's may introduce ambiguous evaluations. The antecedent\n
-LogicHelpText2 = statement is evaluated from left to right, which corresponds to computing\n
-LogicHelpText3 = the innermost variable on the left to the outermost variable on the right.\n
-LogicHelpText4 = To modify this order of evaluation, please insert parenthesis to indicate the
-LogicHelpText5 = logic you want. You may also modify the operators to use. The evaluation
-LogicHelpText6 = logic of 'Mixed' mode is the content of the antecedent field and not the\n
-LogicHelpText7 = operators shown in the rows of the state variable table.
+TypeLogicHint = Escull l'operador l\u00f2gic per unir la variable d'estat amb la precedent (part 'si') d'aquest condicional
+LabelAntecedent = Expressi\u00f3 Antecedent (la part 'si' del Condicional)
+LabelAntecedentHint = Insereix par\u00e8ntesis perqu\u00e8 la sent\u00e8ncia compleixi la l\u00f2gica desitjada
+#HelpButton = Ajuda
+LogicHelpText1 = Barrejar And i Or pot causar avaluacions ambig\u00fces. L'expressi\u00f3\n
+LogicHelpText2 = precedent s'avalua d'esquerra a dreta, com correspon en computaci\u00f3\n
+LogicHelpText3 = de la variable m\u00e9s a l'esquerre fins a la variable m\u00e9s a la dreta.\n
+LogicHelpText4 = Per modificar l'ordre d'avaluaci\u00f3, insereix par\u00e8ntesis per a indicar
+LogicHelpText5 = la l\u00f2gica que vols. Potser has de modificar els operadors a usar. L'avaluaci\u00f3
+LogicHelpText6 = l\u00f2gica d'un mode "Barrejat" \u00e9s el contingut del camp prcedent i no els\n
+LogicHelpText7 = operadors mostrats a les files de la Taula d'Estat de Variable.
 
-# CreateLogixButton = Create Logix
```

Comment:
On 08/26/2016 10:28 AM, Joan de Castro Alemany wrote:
>
> In java/src/jmri/jmrit/beantable/LogixTableBundle_ca.properties 
> <https://github.com/JMRI/JMRI/pull/1804#discussion_r76427434>:
>
> >
> > -# CreateLogixButton = Create Logix
> Can you merge the catalan translatiopns to the Master. When I try to 
> Uoload I can't, so I sync my local to remote repository. Thanks

No, we can't merge this right now.

There are tests failing, and it looks like it may be because you deleted
the contents of

java/src/jmri/jmrit/roster/JmritRosterBundle.properties

Until we fix the tests, we can't merge the PR.

Try running

`git checkout origin master 
java/src/jmri/jmrit/roster/JmritRosterBundle.properties`

Paul